app/views/past_foreign_secretaries/index.html.erb

```diff
@@ -4,10 +4,7 @@
 <header class="govuk-grid-row">
```

Comment:
I've a feeling the breadcrumbs should go above this?  However i'm wondering if this was intentional is the top level and therefore we don't need it

Reply:
You're right, it should  link back to  history like other pages. Soon it will!

Reply:
Fixed.